GenderPayGap.WebUI/Controllers/Admin/AdminOrganisationAddressController.cs

```diff
@@ -150,34 +147,14 @@ private IActionResult OfferNewCompaniesHouseAction(ChangeOrganisationAddressView
                 return SendToManualChangePage(organisation);
             }
 
-            viewModel.ParseAndValidateParameters(Request, m => m.Reason);
-
-            if (viewModel.HasAnyErrors())
-            {
-                viewModel.Organisation = organisation;
-                viewModel.Action = ManuallyChangeOrganisationAddressViewModelActions.OfferNewCompaniesHouseAddress;
-                return View("OfferNewCompaniesHouseAddress", viewModel);
-            }

```

Comment:
I thought about that...
However, the current site behaviour is as follows:

If you are offered a Companies House address, you must click if you want to use it or not (otherwise it gives an error).
If you click no, it works.
If you click yes, only _then_ the reason box is revealed and you must enter a reason as well.

In the model I've added the `GovUkConditionalIf` attribute tor `Reason` for when the user selected 'Yes', so this is fine.